src/main/java/teammates/ui/controller/InstructorCourseInstructorDeleteAction.java

```diff
@@ -27,20 +27,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         new GateKeeper().verifyAccessible(
                 instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR);
 
-        List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
```

Comment:
Hmm, the original algorithm looks like it should be able to prevent the last instructor from being deleted as well. Were you able to replicate the deletion of the last instructor using the original algorithm? Anyhow, the extracted method looks cleaner and more straightforward, good job on that :+1: !